content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -269,8 +269,7 @@ kubeadm only supports Container Network Interface (CNI) based networks (and does
 
 Several projects provide Kubernetes Pod networks using CNI, some of which also
 support [Network Policy](/docs/concepts/services-networking/networkpolicies/). See the [add-ons page](/docs/concepts/cluster-administration/addons/) for a complete list of available network add-ons.
-- IPv6 support was added in [CNI v0.6.0](https://github.com/containernetworking/cni/releases/tag/v0.6.0).
-- [CNI bridge](https://github.com/containernetworking/plugins/blob/master/plugins/main/bridge/README.md) and [local-ipam](https://github.com/containernetworking/plugins/blob/master/plugins/ipam/host-local/README.md) are the only supported IPv6 network plugins in Kubernetes version 1.9.
+- IPv6 support was added in [CNI v0.6.0](https://github.com/containernetworking/cni/releases/tag/v0.6.0). See each plugin's documentation to see if it supports IPv6.
```

Comment:
SGTM.

Reply:
@caseydavenport the existing page doesn't render this correctly. Does the [rendered version](https://deploy-preview-18821--kubernetes-io-master-staging.netlify.com/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/#pod-network) of this look how you wanted?